front/src/applications/operationalStudies/helpers/upsetNewProjectedTrains.ts

```diff
@@ -0,0 +1,39 @@
+import type { TrainSpaceTimeData } from 'applications/operationalStudies/types';
+import { type ProjectPathTrainResult, type TrainScheduleResult } from 'common/api/osrdEditoastApi';
+import { convertIsoUtcToLocalTime } from 'utils/date';
+import type { Dictionary } from 'utils/types';
+
+// TODO: feed the spaceTimeChart with date instead of formatted string
+const formatDatetimeForSpaceTimeChart = (departureTime: string) =>
+  `${convertIsoUtcToLocalTime(departureTime).slice(0, -6)}Z`;
```

Comment:
Perhaps a better way to write this would be:

```ts
new Date().toISOString()
```

Reply:
Hm, by the way, it seems a bit weird to convert to local time and then force UTC without converting to UTC? (The above suggestion performs the conversion.)

Reply:
Yes, it's quite weird, but we have to do so for the spaceTimeChart right now. I would like to fix this weird behaviour right after this PR